services/gossip/adapter/tcp/direct_outgoing.go

```diff
@@ -93,7 +94,8 @@ func (t *DirectTransport) clientHandleOutgoingConnection(ctx context.Context, co
 func (t *DirectTransport) addDataToOutgoingPeerQueue(data *adapter.TransportData, outgoingQueue *transportQueue) {
 	err := outgoingQueue.Push(data)
 	if err != nil {
-		t.metrics.outgoingConnectionSendQueueErrors.Inc()
+		t.metrics.outgoingConnectionSendQueueErrors.Inc() //TODO remove, replaced by following metric
```

Comment:
Remove when? is there an issue?

Reply:
backwards compatibility - I can't assume no one is using this metric